great_expectations/expectations/core/expect_column_quantile_values_to_be_between.py

```diff
@@ -131,16 +131,78 @@ class ExpectColumnQuantileValuesToBeBetween(ColumnExpectation):
     success_keys = (
         "quantile_ranges",
         "allow_relative_error",
+        "auto",
+        "profiler_config",
     )
     default_kwarg_values = {
         "row_condition": None,
-        "allow_relative_error": None,
         "condition_parser": None,
         "quantile_ranges": None,
         "result_format": "BASIC",
         "allow_relative_error": False,
         "include_config": True,
         "catch_exceptions": False,
+        "meta": None,
+        "auto": False,
+        "profiler_config": {
+            "name": "expect_column_quantile_values_to_be_between",  # Convention: use "expectation_type" as profiler name.
+            "config_version": 1.0,
+            "variables": {
+                "quantiles": [
+                    0.25,
+                    0.5,
+                    0.75,
+                ],
+                "allow_relative_error": "linear",
+                "num_bootstrap_samples": 9999,
+                "false_positive_rate": 0.05,
+                "truncate_values": {
+                    "lower_bound": None,
+                    "upper_bound": None,
+                },
+                "round_decimals": 1,
+            },
+            "rules": {
+                "column_quantiles_rule": {
+                    "domain_builder": {
+                        "class_name": "ColumnDomainBuilder",
```

Comment:
Does this domain builder see a pared-down version of the batch with only the column provided?

Reply:
@jcampbell A column-style domain builder gets the active batch.

Reply:
So, how does this rule know to only operate on the user's selected column?

Reply:
@jcampbell The column name is obtained from domain keys of the Expectation at hand and set (using a proper setter method) on the domain builder.  The code for that appears in `Validator` starting line 633.  Thanks.